setup.py

```diff
@@ -1,12 +1,36 @@
 #!/usr/bin/env python
 
-from setuptools import setup
+from setuptools import setup, find_packages
 
-INSTALL_REQUIRES = ["koji", "rpm", "requests", "PyYAML", "simplejson"]
 
-setup(name='alt-src',
-      description='Tool for uploading custom patches to centos source rpm repositories',
-      version='1.0',
-      url='https://github.com/release-engineering/alt-src.git',
-      install_requires=INSTALL_REQUIRES,
-      scripts=['alt-src'])
+def get_description():
+    return "Tool for uploading custom patches to centos source rpm repositories"
+
+
+def get_requirements():
+    with open("requirements.txt") as f:
+        return f.read().splitlines()
+
+
+setup(
+      name="alt-src",
+      version="1.0",
+      packages=find_packages(exclude=["tests"]),
+      url="https://github.com/release-engineering/alt-src",
+      license="GNU General Public License",
+      description=get_description(),
+      classifiers=[
+            "Programming Language :: Python :: 2",
+            "Programming Language :: Python :: 2.4"
+            "Programming Language :: Python :: 2.6",
+            "Programming Language :: Python :: 2.7",
+            "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+      ],
+      install_requires=get_requirements(),
+      python_requires="<=2.7",
```

Comment:
Not sure if we should have this - I understand that the code may not work properly on Python 3 yet, but doesn't this explicitly block even installing it on Python 3, making it hard to test or develop if you're trying to port it?

It's particularly weird as the tests themselves have conditions for dealing with Python3, yet this line should make it completely uninstallable in the first place.